views/js/test/runner/plugins/navigation/next/test.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Test Runner Plugin - next button - warning helper</title>
+    <link rel="stylesheet" type="text/css" href="/tao/views/js/lib/qunit/qunit.css">
+    <script type="text/javascript" src="/tao/views/js/lib/require.js"></script>
+    <script type="text/javascript" src="/tao/views/js/lib/qunit/qunit.js"></script>
+    <script type="text/javascript" src="/tao/views/js/lib/qunit/qunit-parameterize.js"></script>
+    <script type="text/javascript" src="/tao/views/js/lib/blanket/blanket.min.js" data-cover-only="/taoQtiTest/views/js/runner/plugins/navigation/nextWarningHelper.js"></script>
+    <script  type="text/javascript">
+        QUnit.config.autostart = false;
+        require(['/tao/ClientConfig/config'], function(){
+            require(['taoQtiTest/test/runner/plugins/navigation/next/test'], function(){
```

Comment:
If it is related to the helper, should not it be named the same way ? The name of the test refer to the next plugin itself... (`plugins/navigation/next/test`)

It should rather be names: `helper/nextWarningHelper/test`